compiler/noirc_frontend/src/parser/parser.rs

```diff
@@ -61,20 +61,20 @@ pub fn parse_program(source_program: &str) -> (ParsedModule, Vec<ParserError>) {
 
     parsing_errors.extend(lexing_errors.into_iter().map(Into::into));
 
-    (module.unwrap(), parsing_errors)
+    (module.unwrap_or(ParsedModule { items: vec![] }), parsing_errors)
```

Comment:
It would be nice if we could recover a partial module rather than nothing but we can leave that to a later PR.